tools/install_deps/install_bazelisk.sh

```diff
@@ -0,0 +1,23 @@
+# Downloads bazelisk to ${output_dir} as `bazel`.
+date
+
+output_dir=${1:-"/usr/local/bin"}
+
+case "$(uname -s)" in
+    Darwin) name=bazelisk-darwin-amd64 ;;
+    Linux)  name=bazelisk-linux-amd64  ;;
+    *) name=bazelisk-windows-amd64 ;;
+esac
+
+mkdir -p "${output_dir}"
+curl -LO "https://github.com/bazelbuild/bazelisk/releases/download/v1.3.0/${name}"
```

Comment:
Do we want to control bazelisk version from dockerfile env? Or Do we like in the script to have a unique entry?

Reply:
I'm more likely to have a single entry for this, but seems that it's hard to control this in single file or variable in current testing/build architecture.